FusionIIIT/applications/hostel_management/models.py

```diff
@@ -80,22 +80,18 @@ class HallWarden(models.Model):
     def __str__(self):
         return str(self.hall) + '  (' + str(self.faculty.id.user.username) + ')'
     
-
-class GuestRoomDetail(models.Model):
+class HallAdmin(models.Model):
     """
-    Records information related to guest rooms in Hall of Residences.
+    Records the admin of Hall of Residences.
 
     'hall' refers to the related Hall of Residence.
-    'room_no' stores the guest room number.
-    'room_status' stores the current status of the guest room from the available choices in 'ROOM_STATUS'.
+    'faculty' refers to the acting admin for the same Hall of Residence.
     """
     hall = models.ForeignKey(Hall, on_delete=models.CASCADE)
-    room_no = models.CharField(max_length=4, unique=True)
-    room_status  = models.CharField(max_length=20, choices=HostelManagementConstants.ROOM_STATUS, default='Available')
+    faculty = models.ForeignKey(Faculty, on_delete=models.CASCADE)
```

Comment:
if this is not a new model use NULL=true, otherwise it's fine.

Reply:
Resolved